tools/c7n_azure/tests/azure_common.py

```diff
@@ -179,6 +179,8 @@ def _response_callback(self, response):
                                response['headers'].items()
                                if k.lower() not in self.FILTERED_HEADERS}
 
+        response['headers']['content-location'] = [AzureVCRBaseTest._replace_subscription_id(cl) for cl in response['headers']['content-location']]
```

Comment:
Does this header actually do anything for us?  Another option is to just add it to the filtered header list at the top of the file so it disappears.  This seems fine too though.

Reply:
Good point. Filtered it out.